aten/src/ATen/native/AutogradComposite.cpp

```diff
@@ -4,12 +4,8 @@
 namespace at {
 namespace native {
 
-// _make_dual is actually implemented in VariableTypeManual.cpp, this is just a stub
```

Comment:
I think we could leave a comment here saying that this should only be called in inference mode.